### PR TITLE
Deal with all skipped tests #closes 204

### PR DESCRIPTION
**Pull request by mikedao**
* Removed all admin tests that have been rewritten and are duplicates.
* Removed old tests for functionality we are not implementing.
* Removed projects_helper method that wasn't being used.

100% tests passing, 98.71% test coverage.

### CODE REVIEW

File: app/controllers/categories_controller.rb [hunk 1]
```diff
@@ -6,6 +6,5 @@ def index
 
   def show
     @category = Category.find_by(id: params[:id])
-    @projects = @category.projects
   end
```
Comment (EmilyMB) [inline app/controllers/categories_controller.rb]: I think we can delete this show method and all reference to category show paths.

File: app/helpers/projects_helper.rb [hunk 2]
```diff
@@ -1,9 +1,2 @@
 module ProjectsHelper
-  def add_category(categories)
-    categories.map do |category_id|
-      Category.find_by(:id => category_id)
-    end.compact.each do |category|
-      @project.categories << category
-    end
-  end
 end
```
Comment (EmilyMB) [inline app/helpers/projects_helper.rb]: Why not delete the whole file?

File: app/views/categories/show.html.erb [hunk 3]
```diff
@@ -1,20 +1,4 @@
...
-      <h5 class="project-desc"><%= project.description %></h5>
-      <%= form_for :pending_loan, url: pending_loan_path do |f| %>
-        <%= f.hidden_field :project_id, value: project.id %>
-        <%= f.hidden_field :loan_amount, value: project.price %>
-        <%= f.submit 'Lend', class: "btn btn-default" %>
-      <% end %>
-    </div>
-    <% end %>
-  </div>
 </div>
```
Comment (EmilyMB) [inline app/views/categories/show.html.erb]: Why not delete the whole file?

File: app/controllers/categories_controller.rb (hunk 1 above)
Comment (mikedao) [inline app/controllers/categories_controller.rb]: I'm leaving it in place as an emergency plan B for Orion and Jeff.

File: test/integration/admin_login_test.rb [hunk 4]
```diff
@@ -116,183 +106,4 @@ class AdminUserTest < ActionDispatch::IntegrationTest
...
-    order = create(:order, status: "paid")
-    orig_status = order.status
-
-    visit admin_dashboard_path
-    click_link_or_button "Orders"
-    select "completed", from: "update_order_status[status_#{order.id}]"
-    click_button "Update Status"
-
-    refute_equal orig_status, order.reload.status
-  end
```
Comment (EmilyMB) [inline test/integration/admin_login_test.rb]: Trusting that you have written appropriate tests to replace these :smile:

File: app/helpers/projects_helper.rb (hunk 2 above)
Comment (mikedao) [inline app/helpers/projects_helper.rb]: I will look into doing this. Checking for dependencies.

File: app/views/categories/show.html.erb (hunk 3 above)
Comment (mikedao) [inline app/views/categories/show.html.erb]: Leaving this in place for Orion and Jeff.

File: test/integration/admin_login_test.rb (hunk 4 above)
Comment (mikedao) [inline test/integration/admin_login_test.rb]: It's in the admin tests.

File: app/helpers/projects_helper.rb (hunk 2 above)
Comment (mikedao) [inline app/helpers/projects_helper.rb]: Removed.

File: app/helpers/projects_helper.rb [hunk 5]
```diff
@@ -1,9 +0,0 @@
-module ProjectsHelper
-  def add_category(categories)
-    categories.map do |category_id|
-      Category.find_by(:id => category_id)
-    end.compact.each do |category|
-      @project.categories << category
-    end
-  end
-end
```
Comment (EmilyMB) [inline app/helpers/projects_helper.rb]: :heart: the cleanup

**Comment by EmilyMB** [comment]
It's great to see all the tests passing with no skipped ones :smile:

**Pull request merged**